src/BIP32.php

```diff
@@ -64,6 +64,13 @@ class BIP32
     public static $litecoin_testnet_public = '0436f6e1';
     public static $litecoin_testnet_private = '0436ef7d';
     public static $litecoin_testnet_version = '6f';
+    // Monacoin
+    public static $monacoin_mainnet_public = '';
+    public static $monacoin_mainnet_private = '';
+    public static $monacoin_mainnet_version = '32';
+    public static $monacoin_testnet_public = '';
+    public static $monacoin_testnet_private = '';
+    public static $monacoin_testnet_version = '6f';
```

Comment:
Actually, it's not determined officially :(

If you are not willing to add ambiguous constants, I will remove this Monacoin part from my pullreq.
I personally feel that altcoin configuration should be supplied externally, e.g., make `BIP32` class to be an abstract class, and create `BIP32_Bitcoin` class that extends `BIP32` with proper constants defined.